tests/type_inference/test_compress.py

```diff
@@ -1,19 +1,24 @@
 import pytest
 
-import spox.opset.ai.onnx.v17 as op
+import spox.opset.ai.onnx.v17 as op17
+import spox.opset.ai.onnx.v18 as op18
+import spox.opset.ai.onnx.v19 as op19
+import spox.opset.ai.onnx.v20 as op20
 from spox._graph import arguments
 from spox._standard import InferenceError
 from spox._type_system import Tensor
 
 
-def test_compress_inference():
+@pytest.mark.parametrize("op", [op17, op18, op19, op20])
```

Comment:
Why parameterise these test cases specifically? Compress seems the same between these.

Reply:
Good point!